lib/Net/SSLeay.pod

```diff
@@ -1836,7 +1836,7 @@ Converts/exports X509 certificate to string (PEM format).
 
 =item * PEM_get_string_PrivateKey
 
-B<COMPATIBILITY:> not available in Net-SSLeay-1.45 and before
+B<COMPATIBILITY:> not available in Net-SSLeay-1.45 and before. Requires legacy provider with OpenSSL 3.0 and later as long as DES-CBC is the default encryption algorithm.
```

Comment:
```suggestion
B<COMPATIBILITY:> not available in Net-SSLeay-1.45 and before. Requires legacy provider with OpenSSL 3.0 and later if the default value of C<$enc_alg> (C<DES_CBC>) is used.
```

I'd suggest rewording this to make it more user-centric - something like:

> Requires legacy provider with OpenSSL 3.0 and later if the default value of `$enc_alg` (`DES_CBC`) is used.

otherwise it might sound like a discussion of an internal implementation detail.